deploy/platform/kubernetes/features.mk

```diff
@@ -81,3 +81,31 @@ undeploy.feature-kubernetes-monitor:
 	@kubectl delete --ignore-not-found -f https://raw.githubusercontent.com/kubernetes/kube-state-metrics/v2.2.4/examples/standard/cluster-role-binding.yaml
 	@kubectl delete --ignore-not-found -f https://raw.githubusercontent.com/kubernetes/kube-state-metrics/v2.2.4/examples/standard/service.yaml
 	@kubectl delete --ignore-not-found -f https://raw.githubusercontent.com/kubernetes/kube-state-metrics/v2.2.4/examples/standard/deployment.yaml
+
+# @feature: java-agent-injector; use the java agent injector to inject the java agent more natively
+.PHONY: feature-java-agent-injector
+feature-java-agent-injector:
+
+# @feature: java-agent-injector; the swck operator depends on the certificate management of the cert-manager
+.PHONY: install-cert-manager
+install-cert-manager:
+	@kubectl apply -f https://github.com/jetstack/cert-manager/releases/download/v1.3.1/cert-manager.yaml
+	@sh ../../../scripts/wait-cert-manager-ready.sh
+
+# @feature: java-agent-injector; the java agent injector is a component of the swck operator, so we need to deploy the swck operator firstly
+.PHONY: deploy.feature-java-agent-injector
+deploy.feature-java-agent-injector: install-cert-manager
+	@curl -Ls https://dlcdn.apache.org/skywalking/swck/0.5.0/skywalking-swck-0.5.0-bin.tgz | tar -xf - -O ./config/operator-bundle.yaml | kubectl apply -f -
```

Comment:
I think we should keep version/commit id in one space? There is a root .in file controlling this, right?

Reply:
Yep, I will change it.